ros2_batch_job/__main__.py

```diff
@@ -408,7 +473,7 @@ def build_and_test(args, job):
     info("colcon test-result returned: '{0}'".format(ret_test_results))
     print('# END SUBSECTION')
     if args.coverage and args.os == 'linux':
-        process_coverage(args, job)
+        process_coverage(args, job, args.coverage_filter_packages)
```

Comment:
Since `args` are already passed why separately pass `args.coverage_filter_packages`?

Reply:
You are right. Removed in 158f17449c98a746191768783c737c2635940b68